app/controllers/catalog_controller.rb

```diff
@@ -743,14 +743,27 @@ def ot_remove_submit
     if elements.length > 1
       self.x_node = 'root'
     else
-      self.x_node = elements[0].kind_of?(OrchestrationTemplateHot) ? "xx-othot" : "xx-otcfn"
+      if elements[0].kind_of?(OrchestrationTemplateHot)
+        self.x_node = "xx-othot"
+      elsif elements[0].kind_of?(OrchestrationTemplateCfn)
+        self.x_node = "xx-othot"
+      else
+        self.x_node = "xx-otazu"
+      end
     end
     replace_right_cell(nil, trees_to_replace([:ot]))
   end
 
   def ot_add
     assert_privileges("orchestration_template_add")
-    ot_type = x_node == "xx-othot" ? "OrchestrationTemplateHot" : "OrchestrationTemplateCfn"
+    ot_type = case x_node
+      when "xx-othot"
+        "OrchestrationTemplateHot"
+      when"xx-otcfn"
+        "OrchestrationTemplateCfn"
+      else
+        "OrchestrationTemplateAzure"
+    end
```

Comment:
@lgalis I see too many of these lookups in this file.  Suggest 2 conversion methods:
```ruby
def template_to_node_name(object)
  case object
  when OrchestrationTemplateHot
    "xx-othot"
  when OrchestrationTemplateCfn
    "xx-otcfn"
  when OrchestrationTemplateAzure
    "xx-otazu"
  end
end

def node_name_to_class(node_name)
  case node_name
  when "xx-othot"
    OrchestrationTemplateHot
  when "xx-otcfn"
    OrchestrationTemplateCfn
  when "xx-otazu"
    OrchestrationTemplateAzure
  end
end
```